integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataLoadRDD.scala

```diff
@@ -462,3 +478,155 @@ class CarbonDataLoadRDD[K, V](
   }
 }
 
+/**
+ * Use this RDD class to load RDD
+ * @param sc
+ * @param result
+ * @param carbonLoadModel
+ * @param storeLocation
+ * @param hdfsStoreLocation
+ * @param kettleHomePath
+ * @param columinar
+ * @param loadCount
+ * @param tableCreationTime
+ * @param schemaLastUpdatedTime
+ * @param prev
+ * @tparam K
+ * @tparam V
+ */
+class CarbonRDDDataLoadRDD[K, V](
```

Comment:
suggest to change to `DataFrameLoaderRDD`, it means the RDD to load the dataframe into carbon.
change `CarbonDataLoadRDD` to `DataFileLoaderRDD` or 'CSVFileLoaderRDD`. It matches loadDataFrame and loadDataFile function name accordinality.